crates/swc/tests/tsc-references/privateNameAndObjectRestSpread_es5.2.minified.js

```diff
@@ -13,14 +13,7 @@ var _prop = new WeakMap(), C = function() {
         });
     }
     return C.prototype.method = function(other) {
-        var obj = _object_spread({}, other);
-        _class_private_field_get(obj, _prop);
-        var rest = _extends({}, other);
-        _class_private_field_get(rest, _prop);
-        var statics = _object_spread({}, C);
-        _class_static_private_field_spec_get(statics, C, _propStatic);
-        var sRest = _extends({}, C);
-        _class_static_private_field_spec_get(sRest, C, _propStatic);
+        _class_private_field_get(_object_spread({}, other), _prop), _class_private_field_get(_extends({}, other), _prop), _class_static_private_field_spec_get(_object_spread({}, C), C, _propStatic), _class_static_private_field_spec_get(_extends({}, C), C, _propStatic);
```

Comment:
This is incorrect, since e.g. calling `_class_private_field_get` might change the exported value of  `_class_private_field_get`. Now the read and the call are switched.

Reply:
Yeah, I need to make it abort on imports, as we can't analyze those files.
Thank you!